pkgs/applications/window-managers/sway/lock-effects.nix

```
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub
+, meson, ninja, pkgconfig, scdoc
+, wayland, wayland-protocols, libxkbcommon, cairo, gdk-pixbuf, pam
+}:
+
+stdenv.mkDerivation rec {
+  pname = "swaylock-effects";
+  version = "1.6-0";
+
+  src = fetchFromGitHub {
+    owner = "mortie";
+    repo = "swaylock-effects";
+    rev = "v${version}";
+    sha256 = "11nap4xylq48pv7qaqxdcz3lbb2hyki8r996rzfzpw467mkxhsr5";
```

```suggestion
    sha256 = "15lshqq3qj9m3yfac65hjcciaf9zdfh3ir7hgh0ach7gpi3rbk13";
```

Looks like the hash changed, causing build to fail.